src/codeu/chat/client/Controller.java

```diff
@@ -17,6 +17,7 @@
 import java.io.PrintWriter;
 import java.io.StringWriter;
 import java.lang.Thread;
+import codeu.chat.util.Time; // added
```

Comment:
I don't typically see comments on import statements. Could you clarify what you mean in the comment? Also, I don't see the Time class being used here?